Main/src/Coffee/Action.java

```diff
@@ -0,0 +1,186 @@
+package Coffee;
+
+import java.util.Scanner;
+
+public class Action {
+    public static int water=400;
+    public static int milk=540;
+    public static int coffeeBeans=120;
+    public static int dcups=9;
+    public static int money=550;
+
+    public static void getAction() {
+        System.out.println("Step 1: Waiting for the command");
+        Scanner in = new Scanner(System.in);
+        System.out.println("Please enter required action: 'buy', 'fill', 'pull', 'remaining' or 'exit'...");
+        String actionType = in.nextLine();
+        switch(actionType) {
+            case "fill":
+                Action.fillMachine();
+                break;
+            case "pull":
+                Action.pull();
+                break;
+            case "buy":
+                Action.orderCoffee(in);
+                break;
+            case "remaining":
+                Action.machineStatus();
+                break;
+            case "exit":
+                break;
+        }
+}
+
+    public static void orderCoffee(Scanner in){
+        System.out.println("Step 2: Waiting for the order");
+        System.out.println("Please input coffee type ");
+        System.out.println("1. Espresso");
+        System.out.println("2. Latte");
+        System.out.println("3. Cappuccino");
+        System.out.println("or print 'back' to return... ");
+        String coffeeType = in.nextLine();
+
+
+        switch(coffeeType) {
+            case "Espresso":
+                System.out.println("Please input cups quantity ");
+                int cup = in.nextInt();
+                Action.espresso(cup);
+                break;
+            case "Latte":
+                System.out.println("Please input cups quantity ");
+                cup = in.nextInt();
+                Action.latte(cup);
+                break;
+            case "Cappuccino":
+                System.out.println("Please input cups quantity ");
+                cup = in.nextInt();
+                Action.cappuccino(cup);
+                break;
+            case "back":
+                getAction();
+                in.close();
+                break;
+        }
+
+        in.close();
+    }
+
+        public static void fillMachine()
+    {  stepOne: ;
```

Comment:
@GayaneGulanyan jan what is this ?

Reply:
Than was the try of using breach + label, but now it is fixed using case